src/Symfony/Component/DependencyInjection/ContextualizedContainerBuilderInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection;
+
+/**
+ * Implemented by a ContainerBuilder sharing a context during build time.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+interface ContextualizedContainerBuilderInterface
```

Comment:
At first it was named `ContextualizedContainerInterface`.
I've no strong opinion on this, but as I already mention the `Context` is used while building the container, and for given reasons pointed up by @chalasr , I changed to `ContextualizedContainerBuilderInterface`.